### PR TITLE
Nuxt 3 "v3" branch

### PR DESCRIPTION
**Pull request by nathanchase**
This is an initial commit of a Nuxt 3 branch of the dayjs module, based off work done at [https://github.com/nathanchase/dayjs](https://github.com/nathanchase/dayjs).

Based on [https://github.com/nuxt-community/dayjs-module/issues/399](https://github.com/nuxt-community/dayjs-module/issues/399), I've created this v3 branch.

See https://github.com/nuxt-community/dayjs-module/issues/376 for more info.

### CODE REVIEW

**Comment by potato4d** [comment]
@nathanchase 
I appreciate your PR!
I would like to work on the `next` branch until Nuxt v3 is stable.

This PR I will check during this weekend. Please wait a while ;)

**Comment by nathanchase** [comment]
> @nathanchase I appreciate your PR! I would like to work on the `next` branch until Nuxt v3 is stable.
> 
> This PR I will check during this weekend. Please wait a while ;)

No problem! I wasn't sure how much work was underway on the Nuxt 3 version so thought I'd just share this with you in case it was helpful.

Obviously, it's based on an entirely different repo, so it's more just so you can take a look at it in isolation and see what's worth pulling in.

File: .npmignore [hunk 1]
```diff
@@ -1,4 +0,0 @@
-src/
```
Comment (potato4d) [inline .npmignore:1]: I would like to see this remain as well.
If it is too difficult, I will do it.

File: .prettierignore [hunk 2]
```diff
@@ -1,5 +0,0 @@
-.nuxt/
```
Comment (potato4d) [inline .prettierignore:1]: I would like to see this remain as well.
If it is too difficult, I will do it.

File: .prettierrc [hunk 3]
```diff
@@ -1,4 +0,0 @@
-{
```
Comment (potato4d) [inline .prettierrc:1]: I would like to see this remain as well.
If it is too difficult, I will do it.

File: .versionrc.js [hunk 4]
```diff
@@ -1,11 +0,0 @@
-module.exports = {
```
Comment (potato4d) [inline .versionrc.js:1]: I would like to see this remain as well.
If it is too difficult, I will do it.

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,257 +1,23 @@
-# Changelog
```
Comment (potato4d) [inline CHANGELOG.md:1]: I would like to see this remain as well.
If it is too difficult, I will do it.

File: codecov.yml [hunk 6]
```diff
@@ -1,7 +0,0 @@
-comment:
```
Comment (potato4d) [inline codecov.yml:1]: I would like to see this remain as well.

File: renovate.json [hunk 7]
```diff
@@ -1,10 +0,0 @@
-{
```
Comment (potato4d) [inline renovate.json:1]: I would like to see this remain as well.

File: pnpm-lock.yaml [hunk 8]
```diff
@@ -0,0 +1,6082 @@
+lockfileVersion: 5.4
```
Comment (potato4d) [inline pnpm-lock.yaml:1]: I would like to request a change to the use of yarn.

File: yarn.lock [hunk 9]
```diff
@@ -1,12873 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (potato4d) [inline yarn.lock:1]: We would like to request a change to the use of yarn.

**Comment by potato4d** [comment]
> Obviously, it's based on an entirely different repo, so it's more just so you can take a look at it in isolation and see what's worth pulling in.

I see, I beg your pardon for that! I'll take this as a reference and cherry-pick what I need from the current master branch :)

**Review by potato4d — APPROVED** [review]
@nathanchase

For the time being, I would like to merge this PR to mix with the current content.
Depending on the review, I may request an additional pull request.

**Pull request merged**